data/mods/Xedra_Evolved/design_doc_spoilers.md

```diff
@@ -0,0 +1,175 @@
+- Hedge magick requires either components, particular conditions like a time of day or location, or (often) both.
+- Hedge magick requires extensive study to learn. Most hedge magick the survivor is likely to find developed from first principles by recluses and weirdos, and the survivor has to interpret those notes into a decipherable magickal formula.
+- Existing hedge magick was often developed by looking at real-world occultism, but this is less importan than keeping hedge magick's power level low--real-world occultists make some absolutely wild claims of their power.
+
+Alchemy: TBD
+
+## Vampires
+
+Xedra Evolved vampires are not walking corpses who drink the blood of the living. At least, not at first. Vampirism is a progressive disease and many sufferers are more alive than they are dead. While they must drink blood and can derive power from it, they still need to eat, drink, and sleep just like mortals do. Becoming an actual undead monster requires a conscious decision on the part of the nascent vampire and can only be accomplished by committing heinous crimes (at the time of this writing, murdering a dear and trusting friend). Actual vampires are monsters without exception, predators who have extinguished all feeling from their hearts. 
+
+Vampires have existed for a long time on the Earth of Xedra Evolved, and on some--though not all--extradimensional Earths as well. It is possible that humanity's first steps into agriculture and gathering into stationary cities were encouraged by vampires, so that they would have an easier time maintaining their herds. They use their `renfields`, brainwashed mortal agents, to manipulate human society and keep themselves safe from those who would destroy them. 
+
+Vampires require human blood, or the blood of human-like species. They cannot survive on animal blood, so they are forced to prey on fellow sapients to survive. They also cannot drink vampire blood (though see below), because vampires have enough control over their own blood to prevent another from drinking it. A vampire who does not use any of their powers (called Blood Arts) is capable of surviving for quite a long time without needing to feed. 
+
+Vampires gain new powers with time, intended to mimic the progression of a disease.
+
+### Blood Arts
+
+Blood Arts is the term for vampires special abilities. With then, vampires can become stronger, faster, more resilient, have better senses, hypnotize or charm their prey, and eventually turn into wolf, bat, or mist.  More powerful vampires can also control darkness and shadows. The balance point for bLood arts is intentionally set so that a vampires who uses several of them will either have to feed soon after to avoid the `withering` (the status penalties that come from being low on blood), or will have to gorge beforehand and become `blood_quenched` in order to have enough blood to use all the Blood Arts they want during combat.
```

Comment:
```suggestion
Blood Arts is the term for vampires special abilities. With them, vampires can become stronger, faster, more resilient, have better senses, hypnotize or charm their prey, and eventually turn into wolf, bat, or mist.  More powerful vampires can also control darkness and shadows. The balance point for blood arts is intentionally set so that a vampires who uses several of them will either have to feed soon after to avoid the `withering` (the status penalties that come from being low on blood), or will have to gorge beforehand and become `blood_quenched` in order to have enough blood to use all the Blood Arts they want during combat.
```